code/modules/mob/living/simple_animal/friendly/drone/_drone.dm

```diff
@@ -277,7 +277,7 @@
 	// Why would bees pay attention to drones?
 	return 1
 
-/mob/living/simple_animal/drone/electrocute_act(shock_damage, obj/source, siemens_coeff = 1, safety = 0, tesla_shock = 0, illusion = 0, stun = TRUE)
+/mob/living/simple_animal/bot/secbot/electrocute_act(shock_damage, source, siemens_coeff = 1, flags = NONE)
```

Comment:
this... confuses me. why bot/secbot..?

Reply:
I was very tired when I made this PR. Bad copypasta. :sweat_smile: